src/lib.zig

```diff
@@ -848,6 +854,12 @@ const asn1 = struct {
     }
 };
 const bearssl_sources = [_][]const u8{
+    "/BearSSL/tools/keys.c",
+    "/BearSSL/tools/files.c",
+    "/BearSSL/tools/names.c",
+    "/BearSSL/tools/xmem.c",
+    "/BearSSL/tools/errors.c",
+    "/BearSSL/tools/vector.c",
```

Comment:
I think these should be without leading /

Reply:
oops. indeed. fixed with https://github.com/stef/zig-bearssl/commit/54a192bee8fdddc94438011f9987b1b067f55d06